ethereum/contracts/bridgehub/bridgehub-deps/BridgehubStorage.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (proxy/Proxy.sol)
+
+pragma solidity ^0.8.13;
+
+// import "../../state-transition/state-transition-interfaces/IStateTransition.sol";
+
+import {IVerifier, VerifierParams} from "../../state-transition/chain-interfaces/IVerifier.sol";
+// import "../../state-transition/Verifier.sol";
+import {UpgradeStorage} from "../../state-transition/chain-deps/StateTransitionChainStorage.sol";
+import "../../common/interfaces/IAllowList.sol";
+import "../../common/Messaging.sol";
+import "../../state-transition/libraries/PriorityQueue.sol";
+
+struct BridgehubStorage {
+    /// @dev Storage of variables needed for deprecated diamond cut facet
+    uint256[7] __DEPRECATED_diamondCutStorage;
+    /// @notice Address which will exercise critical changes to the Diamond Proxy (upgrades, freezing & unfreezing)
+    address governor;
+    /// @notice Address that the governor proposed as one that will replace it
+    address pendingGovernor;
+    /// @notice List of permitted validators
+    mapping(address => bool) validators;
+    /// @dev Verifier contract. Used to verify aggregated proof for batches
+    IVerifier verifier;
+    /// @notice Total number of executed batches i.e. batches[totalBatchesExecuted] points at the latest executed batch
+    /// (batch 0 is genesis)
+    uint256 totalBatchesExecuted;
```

Comment:
These fields are deprecated, right?

Reply:
Bridgehub now TUP, not relevant ( was needed for backwards compatibility)